src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -57,15 +58,24 @@ class SidebarScreen extends Component {
         Performance.markStart(CONST.TIMING.SIDEBAR_LOADED);
         Timing.start(CONST.TIMING.SIDEBAR_LOADED, true);
 
-        if (this.props.isFirstTimeNewExpensifyUser) {
-            // For some reason, the menu doesn't open without the timeout
-            setTimeout(() => {
-                this.toggleCreateMenu();
-
-                // Set the NVP back to false (this may need to be moved if this NVP is used for anything else later)
+        // NOTE: This setTimeout is required due to a bug in react-navigation where modals do not display properly in a drawerContent
+        // This is a short-term workaround, see this issue for updates on a long-term solution: https://github.com/Expensify/App/issues/5296
+        setTimeout(() => {
+            if (this.props.isFirstTimeNewExpensifyUser) {
+                const hasFreePolicy = _.chain(this.props.policies)
```

Comment:
We reverted this code in https://github.com/Expensify/App/pull/5360/files#diff-9b9fee594bc95324fa21caa02ac4d6d9692ce460c56587ff9aea35893fb3dde2L82, is this up to date?

Reply:
Sorry, bad merge. Will fix and retest.